website/docs/r/rds_cluster.html.markdown

```diff
@@ -123,7 +123,7 @@ The following arguments are supported:
 * `port` - (Optional) The port on which the DB accepts connections
 * `preferred_backup_window` - (Optional) The daily time range during which automated backups are created if automated backups are enabled using the BackupRetentionPeriod parameter.Time in UTC. Default: A 30-minute window selected at random from an 8-hour block of time per region. e.g. 04:00-09:00
 * `preferred_maintenance_window` - (Optional) The weekly time range during which system maintenance can occur, in (UTC) e.g. wed:04:00-wed:04:30
-* `replication_source_identifier` - (Optional) ARN of a source DB cluster or DB instance if this DB cluster is to be created as a Read Replica.
+* `replication_source_identifier` - (Optional) ARN of a source DB cluster or DB instance if this DB cluster is to be created as a Read Replica. If DB Cluster is part of a Global Cluster, use the [`lifecycle` configuration block `ignore_changes` argument](/docs/configuration/resources.html#ignore_changes) to prevent Terraform from showing differences for this argument instead of configuring this value.
```

Comment:
did you come across the behavior where setting this argument as `Computed` would cause the "secondary" cluster to remain even after successfully destroying its instance and the other clusters, but no error would be returned?  curious if its different than the case of the `global_cluster_identifier`, where having that field as `Computed` doesn't allow deletion from the global_cluster at all

Reply:
The crux of the issue with `global_cluster_identifier` was that it was essentially not possible to perform a configuration update to remove the cluster from the Global Cluster while also keeping the Global Cluster around. There's small writeup of the problem with `Computed: true` not even allowing operators to set the value to `""` to remove things in the https://github.com/terraform-providers/terraform-provider-aws/pull/14487 PR description. I tried fiddling with it last night for a few hours without luck, since Terraform never invokes the resource update function even though it should.

I do not believe there is a covering test for removing `replication_source_identifier` at the moment (to promote a secondary cluster to standalone) like there is `TestAccAWSRDSCluster_GlobalClusterIdentifier_EngineMode_Global_Remove` 🙁  It'd be awesome to add one though! It'd be related to https://github.com/terraform-providers/terraform-provider-aws/issues/6749.